modules/tensor_mechanics/doc/content/source/materials/HillElastoPlasticityStressUpdate.md

```diff
@@ -35,6 +35,21 @@ The model currently uses power law hardening:
 
 where $K$ is the hardening constant and $n$ is the hardening exponent. The hardening exponent has a default value of 1.0 and this value can be modified by the user by setting the parameter [!param](/Materials/HillElastoPlasticityStressUpdate/hardening_exponent). The [!param](/Materials/HillElastoPlasticityStressUpdate/yield_stress) ($\sigma_{yo}$) and [!param](/Materials/HillElastoPlasticityStressUpdate/hardening_constant) ($K$) are the required parameters to be supplied by the user.
 
+## Usage for 3D models
+
+When this model is used for curved 3D geometry (currently limited to only cylindrical geometry) with anisotropic plasticity, the model requires rotation of the stresses (global coordinate system (Cartesian) to local coordinate system (cylindrical)) and strains (local to global coordinate system). For cylindrical geometry, the user should set the parameter [!param](/Materials/HillElastoPlasticityStressUpdate/local_cylindrical_csys) to 'true' along with [!param](/Materials/HillElastoPlasticityStressUpdate/axis) (= x for x-axis y for y-axis and z for z-axis) used as axis of rotation. The option for using this material model for 3D spherical geometry will be added sometime in the future.
+
+The stress and strain tensors $T$ are rotated according to the following equation:
+
+\begin{equation}
+[T'] = [Q'] [T] [Q_{transpose}]
+\end{equation}
+
+where $T$ is a tensor in the old coordinate system and $T'$ is the tensor in the new coordinate system. The square brackets $[$ $]$ denote matrix form (3x3) of the tensor. The basis vectors of old coordinate system should be arranged column wise to construct the rotation matrix $Q$.
+
+!alert warning
+This model has not been tested for a spherical geometry; a test for spherical geometry will be added sometime in the future. At present, using this model for plastically anisotropic material in 3D spherical geometry shape is not recommended.
```

Comment:
You removed this feature, so I'd remove the warning here too.

Reply:
Good catch. Removed.